projects/docs-tools/src/modules/type-definitions/typedoc-adapter.service.spec.ts

```diff
@@ -212,13 +212,7 @@ describe('TypeDoc adapter', () => {
             {
               name: '__get',
               comment: {
-                shortText: 'The foo of the FooClass.',
```

Comment:
The moving of the default in `tags` here was because that is how TypeDoc actually denotes this. The test was incorrect compared to what TypeDoc does